Controls/Cafe/Customer2.c

```diff
@@ -10,7 +11,11 @@
  * revenue and print the current business revenue.
  */
 void Customer2_checkout(char** name, int* cost) {
-	OS_ERR err; // Make sure to check for errors and print the error code if not OS_ERR_NONE
+	*name = menu[0]; // Set val of name to menu[0]
+	*cost = costs[0]; // Set val of cost to costs[0]
+	printf("\nCurrent business revenue: $%d\n", revenue); // Print revenue before purchase
+	revenue += *cost; // Add cost of item to revenue
```

Comment:
This revenue variable needs to be protected, += is not atomic, and is a read-modify-write condition